db/seeds.rb

```diff
@@ -14,5 +14,6 @@
 # Create the default user account.
 user = UserEmail.find_by_email('test@example.org')
 unless user
-  User.create!(email: 'test@example.org', password: 'Coursemology!', role: :administrator)
+  User.create!(name: 'Administrator', email: 'test@example.org',
```

Comment:
Well if we want to be consistent strictly speaking we should be placing one parameter per line... meh